packages/models-library/src/models_library/api_schemas_dynamic_sidecar/telemetry.py

```diff
@@ -56,29 +55,13 @@ class DiskUsage(BaseModel):
     free: ByteSize = Field(description="remaining space")
 
     total: ByteSize = Field(description="total space = free + used")
-    used_percent: NonNegativeFloat = Field(
+    used_percent: float = Field(
```

Comment:
is nonnegativefloat not anymore existing?

Reply:
I guess you guys talked about this  but not sure what the outcome was :-)